prime-router/src/test/kotlin/tokens/TokenAuthenticationTests.kt

```diff
@@ -330,4 +430,184 @@ class TokenAuthenticationTests {
         assertFalse(Scope.scopeListContainsScope("x   x", ""))
         assertFalse(Scope.scopeListContainsScope("x   x", " "))
     }
+
+    @Test
+    @Ignore
```

Comment:
This looks like an awesome test, but we'll need to move it to TestReportStream,kt -- I had to mark this as @Ignore unfortunately because it won't pass when ReportStream itself is not running.

Reply:
fixed.